packages/flutter/lib/src/material/text_field.dart

```diff
@@ -298,6 +297,12 @@ class TextField extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.enableInteractiveSelection}
   final bool enableInteractiveSelection;
 
+  /// Whether or not this field allows text selection.
```

Comment:
This isn't quite right, since it's about interactive text selection; you can still programmatically change the selection with the controller.

Sadly, I think it will be necessary to exhaustively explain what this getter means vis. enableInteractiveSelection and obscureText.

Similarly, enabledInteractiveSelection will need to explain its (new) default value, etc.